pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -40,6 +40,7 @@
   social_hoc2020_global_movement: 'The Hour of Code is a global movement reaching tens of millions of students. One-hour tutorials are available in 45+ languages for all ages.'
   social_hoc2020_coming_dates: 'The Hour of Code is coming December 7-13, 2020. Computer science is foundational for every student to learn.'
   hoc2020_ai_subheader: 'The Hour of Code 2020 is coming!'
+  hoc2020_ai_subheader_actual_hoc: 'The Hour of Code 2020 is here!'
```

Comment:
note that we will not be able to get this string translated in time for HOC

Reply:
👍 we are aware

Reply:
we ended up using an existing string and doing away with the '2020' so it will be translated.